network_importer/adapters/netbox_api/adapter.py

```diff
@@ -409,7 +414,8 @@ def get_intf_from_netbox(self, device_name, intf_name):
             return False
 
         intf = self.interface(name=intf_name, device_name=device_name, remote_id=intfs[0].id)
-
+        intf = apply_model_flag(intfs, intf)
```

Comment:
I think `intfs` should be `intfs[0]` here, otherwise we are passing a list and not a pynetbox object

Reply:
Thanks, updated.